app/jobs/fastly_log_processor.rb

```diff
@@ -0,0 +1,75 @@
+require 'zlib'
+FastlyLogProcessor = Struct.new(:bucket, :key) do
```

Comment:
> I dont like Structs. they are usually slower than normal classes too. Can we define a class here instead?

I was following app/jobs/notifier.rb as an example (which uses a Struct), and the Struct saves me some boilerplate initializer/accessor code.

I was aiming to be as conventional as possible. :smile: @arthurnn Is the project's convention now to use classes?